pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -785,6 +785,25 @@ func (p *criStatsProvider) addCadvisorContainerStats(
 		cs.UserDefinedMetrics = cadvisorInfoToUserDefinedMetrics(caPodStats)
 	}
 
+	cpu, memory := cadvisorInfoToCPUandMemoryStats(caPodStats)
+	if cpu != nil {
+		cs.CPU = cpu
+	}
+	if memory != nil {
+		cs.Memory = memory
+	}
+	accelerators := cadvisorInfoToAcceleratorStats(caPodStats)
+	cs.Accelerators = accelerators
+}
+
+func (p *criStatsProvider) addCadvisorContainerCPUAndMemoryStats(
```

Comment:
so the test for this functionality mocks cAdvisor results. Can you confirm that cAdvisor that was used in 1.19 was producing these metrics? Just a review of commits diff may be enough I think

Reply:
We have actually tested this end-to-endly, and have verified that accelerator metrics are available thru summary API